vello_tests/Cargo.toml

```diff
@@ -16,3 +16,7 @@ anyhow = { workspace = true }
 pollster = { workspace = true }
 png = "0.17.13"
 futures-intrusive = { workspace = true }
+nv-flip = "0.1.2"
+image = { workspace = true, features = ["png"] }
+
+scenes = { workspace = true }
```

Comment:
If you're adding a dependency to the `scenes` example, you might at well make it into a full `vello_test_scenes` root crate. Having a crate depend on an example is somewhat confusing.

Reply:
I'm hesitant, mostly because of the assets directory. Specifically, the issue is:
1) `vello_test_scenes` makes use of the `examples/assets`.
2) I think it's clearer to have `assets` under `examples`, because that's the main route through which users use them
3) I don't think a top-level `vello_test_scenes` should be using assets in the `examples/assets` folder.

Reply:
What's wrong with having assets in vello_test_scenes?

Reply:
- The existing downloads folder is under `examples/assets` and is `.gitignore`d, which means that moving that folder will require manual work from anyone who has been using the downloads folder
- In my opinion, `examples/assets` is a much more natural place to find assets used by the examples than in a test crate - most users who care about examples.
- I would expect users looking for example uses of Vello would be much more likely to look under examples than in a test support crates. Although our entire setup isn't really great for people wanting to find examples.
- I also don't really see that it's worth the noise of having an extra top-level crate for it.

I don't really have a *strong* position on this, but I think leaving it as-is is the least bad option. Happy to change it if you want to impose a different way though :)